integration/smartbft/smartbft_test.go

```diff
@@ -1415,11 +1418,189 @@ var _ = Describe("EndToEnd Smart BFT configuration test", func() {
+			network.Bootstrap()
+
+			var ordererRunners []*ginkgomon.Runner
+			for _, orderer := range network.Orderers {
+				runner := network.OrdererRunner(orderer)
+				runner.Command.Env = append(runner.Command.Env, "FABRIC_LOGGING_SPEC=orderer.consensus.smartbft=debug:grpc=debug:policies=debug")
+				ordererRunners = append(ordererRunners, runner)
+				proc := ifrit.Invoke(runner)
+				ordererProcesses = append(ordererProcesses, proc)
+				Eventually(proc.Ready(), network.EventuallyTimeout).Should(BeClosed())
+			}
+
+			joinChannel(network, channel)
+
+			By("Waiting for followers to see the leader")
+			Eventually(ordererRunners[1].Err(), network.EventuallyTimeout, time.Second).Should(gbytes.Say("Message from 1"))
+			Eventually(ordererRunners[2].Err(), network.EventuallyTimeout, time.Second).Should(gbytes.Say("Message from 1"))
+			Eventually(ordererRunners[3].Err(), network.EventuallyTimeout, time.Second).Should(gbytes.Say("Message from 1"))
+
+			leader := network.Orderers[0]
+			follower := network.Orderers[1]
+
+			By("Create a GRPC")
+			grpcConn := network.NewClientConn(
+				network.OrdererAddress(follower, nwo.ClusterPort),
+				filepath.Join(network.OrdererLocalTLSDir(follower), "ca.crt"),
+				filepath.Join(network.OrdererLocalTLSDir(leader), "server.crt"),
+				filepath.Join(network.OrdererLocalTLSDir(leader), "server.key"),
+			)
+			defer grpcConn.Close()
+
+			By("Create a step client")
+			clusterNodeServiceClient := ordererProtos.NewClusterNodeServiceClient(grpcConn)
+			stepClient, err := clusterNodeServiceClient.Step(context.Background())
+			Expect(err).NotTo(HaveOccurred())
+
+			signer := &nwo.SigningIdentity{
+				CertPath: network.OrdererSignCert(leader),
+				KeyPath: filepath.Join(
+					network.OrdererLocalCryptoDir(leader, "msp"),
+					"keystore",
+					"priv_sk",
+				),
+				MSPID: network.Organization(leader.Organization).MSPID,
+			}
+
+			client := &cluster.NodeClientStream{
+				Version:           0,
+				StepClient:        stepClient,
+				SourceNodeID:      uint64(leader.Id),
+				DestinationNodeID: uint64(follower.Id),
+				Signer:            signer,
+				Channel:           channel,
+			}
+
+			By("Authenticate to the leader step service")
+			err = client.Auth()
+			Expect(err).NotTo(HaveOccurred())
+
+			By("Create an envelope with a valid signature")
+			env, err := protoutil.CreateSignedEnvelope(
+				common.HeaderType_ENDORSER_TRANSACTION,
+				channel,
+				signer,
+				&common.Envelope{Payload: []byte("TEST_MESSAGE")},
+				0,
+				0,
+			)
+			Expect(err).NotTo(HaveOccurred())
+
+			By("Create the consensus request")
+			genesisBlock := network.LoadAppChannelGenesisBlock(channel)
+			req, _ := createPrePrepareRequest(genesisBlock, genesisBlock, env, channel)
+
+			By("Send the request")
+			Expect(client.Send(req)).To(Succeed())
+
+			By("Assert success")
+			Eventually(ordererRunners[follower.Id-1].Err(), network.EventuallyTimeout).Should(gbytes.Say("LogRecord appended successfully"))
```

Comment:
When node 2 sends pre-prepare to node 1 the log of 1 is: 

```
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.341 IDT 0205 DEBU [orderer.consensus.smartbft.chain] HandleMessage -> Message from 2 channel=testchannel1
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.341 IDT 0206 DEBU [orderer.consensus.smartbft.consensus] ProcessMessages -> 1 got message from 2: <PrePrepare with view: 1, seq: 1, payload of 974 bytes, header: MEcCAQEEIO2JmWQbABTpxQPAC3Nthbbu4pic+/pPTB9FaVoSKtdmBCBlv8DoUW6c1Fd78f/bD8WePESdQXiCZWzIe3TryXhcFg==> channel=testchannel1
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.341 IDT 0207 DEBU [orderer.consensus.smartbft.consensus] handleHeartBeat -> Received heartbeat from 2, last heartbeat was 1.999168753s ago channel=testchannel1
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.341 IDT 0208 DEBU [orderer.consensus.smartbft.consensus] processMsg -> 1 got message <PrePrepare with view: 1, seq: 1, payload of 974 bytes, header: MEcCAQEEIO2JmWQbABTpxQPAC3Nthbbu4pic+/pPTB9FaVoSKtdmBCBlv8DoUW6c1Fd78f/bD8WePESdQXiCZWzIe3TryXhcFg==> from 2 with seq 1 channel=testchannel1
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.341 IDT 0209 DEBU [policies] EvaluateSignedData -> == Evaluating *policies.ImplicitMetaPolicy Policy /Channel/Writers ==
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.341 IDT 020a DEBU [policies] EvaluateSignedData -> This is an implicit meta policy, it will trigger other policy evaluations, whose failures may be benign
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.341 IDT 020b DEBU [policies] EvaluateSignedData -> == Evaluating *policies.ImplicitMetaPolicy Policy /Channel/Application/Writers ==
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.341 IDT 020c DEBU [policies] EvaluateSignedData -> This is an implicit meta policy, it will trigger other policy evaluations, whose failures may be benign
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.341 IDT 020d DEBU [policies] EvaluateSignedData -> == Evaluating *cauthdsl.policy Policy /Channel/Application/Org1/Writers ==
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.341 IDT 020e DEBU [policies] SignatureSetToValidIdentities -> signature for identity 0 validated
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.341 IDT 020f DEBU [policies] EvaluateSignedData -> Signature set did not satisfy policy /Channel/Application/Org1/Writers
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.341 IDT 0210 DEBU [policies] EvaluateSignedData -> == Done Evaluating *cauthdsl.policy Policy /Channel/Application/Org1/Writers
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.341 IDT 0211 DEBU [policies] EvaluateSignedData -> == Evaluating *cauthdsl.policy Policy /Channel/Application/Org2/Writers ==
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.342 IDT 0212 DEBU [policies] SignatureSetToValidIdentities -> signature for identity 0 validated
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.342 IDT 0213 DEBU [policies] EvaluateSignedData -> Signature set did not satisfy policy /Channel/Application/Org2/Writers
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.342 IDT 0214 DEBU [policies] EvaluateSignedData -> == Done Evaluating *cauthdsl.policy Policy /Channel/Application/Org2/Writers
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.342 IDT 0215 DEBU [policies] func1 -> Evaluation Failed: Only 0 policies were satisfied, but needed 1 of [ Org1/Writers Org2/Writers ]
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.342 IDT 0216 DEBU [policies] EvaluateSignedData -> Signature set did not satisfy policy /Channel/Application/Writers
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.342 IDT 0217 DEBU [policies] EvaluateSignedData -> == Done Evaluating *policies.ImplicitMetaPolicy Policy /Channel/Application/Writers
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.342 IDT 0218 DEBU [policies] EvaluateSignedData -> == Evaluating *policies.ImplicitMetaPolicy Policy /Channel/Orderer/Writers ==
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.342 IDT 0219 DEBU [policies] EvaluateSignedData -> This is an implicit meta policy, it will trigger other policy evaluations, whose failures may be benign
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.342 IDT 021a DEBU [policies] EvaluateSignedData -> == Evaluating *cauthdsl.policy Policy /Channel/Orderer/OrdererOrg/Writers ==
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.342 IDT 021b DEBU [policies] SignatureSetToValidIdentities -> signature for identity 0 validated
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.342 IDT 021c DEBU [policies] EvaluateSignedData -> Signature set satisfies policy /Channel/Orderer/OrdererOrg/Writers
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.342 IDT 021d DEBU [policies] EvaluateSignedData -> == Done Evaluating *cauthdsl.policy Policy /Channel/Orderer/OrdererOrg/Writers
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.342 IDT 021e DEBU [policies] EvaluateSignedData -> Signature set satisfies policy /Channel/Orderer/Writers
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.342 IDT 021f DEBU [policies] EvaluateSignedData -> == Done Evaluating *policies.ImplicitMetaPolicy Policy /Channel/Orderer/Writers
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.342 IDT 0220 DEBU [policies] EvaluateSignedData -> Signature set satisfies policy /Channel/Writers
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.342 IDT 0221 DEBU [policies] EvaluateSignedData -> == Done Evaluating *policies.ImplicitMetaPolicy Policy /Channel/Writers
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.342 IDT 0222 DEBU [orderer.consensus.smartbft.consensus] verifyPrevCommitSignatures -> Previous proposal verification sequence: 18446744073709551615, current verification sequence: 0 channel=testchannel1
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.342 IDT 0223 INFO [orderer.consensus.smartbft.consensus] verifyPrevCommitSignatures -> Skipping verifying prev commit signatures due to verification sequence advancing from 18446744073709551615 to 0 channel=testchannel1
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.342 IDT 0224 DEBU [orderer.consensus.smartbft.consensus] verifyBlacklist -> DecisionsPerLeader is 0, hence leader rotation is inactive channel=testchannel1
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.363 IDT 0225 DEBU [orderer.consensus.smartbft.chain] append -> LogRecord appended successfully: total size=1160, recordLength=1148, dataCRC=E09F2D39; file=/var/folders/sh/cx2kqdf52bbgygbr8vchj1qc0000gn/T/e2e-smartbft-test842847274/orderers/OrdererOrg.orderer1/etcdraft/wal/testchannel1/0000000000000001.wal, new-offset=1208 channel=testchannel1
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.363 IDT 0226 INFO [orderer.consensus.smartbft.consensus] processProposal -> Processed proposal with seq 1 channel=testchannel1
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.363 IDT 0227 DEBU [orderer.consensus.smartbft.rpc] consensusSent -> Sending msg of 72 bytes to 2 on channel testchannel1 took 13.888µs channel=testchannel1
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.363 IDT 0228 DEBU [orderer.consensus.smartbft.rpc] consensusSent -> Sending msg of 72 bytes to 3 on channel testchannel1 took 3.362µs channel=testchannel1
[e][OrdererOrg.orderer1] 2023-09-13 16:40:30.363 IDT 0229 DEBU [orderer.consensus.smartbft.rpc] consensusSent -> Sending msg of 72 bytes to 4 on channel testchannel1 took 3.696µs channel=testchannel1
```

Reply:
```
Processed proposal with seq 1 channel=testchannel1
```

Maybe try this